easybuild/easyconfigs/m/MFiX/MFiX-24.3-intel-2023b.eb

```diff
@@ -18,19 +18,23 @@ such as drag, polydispersity, attrition, and agglomeration models, among other s
 
 toolchain = {'name': 'intel', 'version': '2023b'}
 toolchainopts = {
-    'usempi': True, 
-    'pic': True, 
-    'extra_fflags': '-fallow-argument-mismatch'
+    'usempi': True,
+    'pic': True,
+    'extra_fflags': '-fallow-argument-mismatch',
 }
 
 sources = ['%(namelower)s-%(version)s.tar.gz']
 checksums = ['d6313d02b4631a6b2eef944c158bb7b1a2eecfe7bcee9f533ce71571f1acffe8']
 
+download_instructions = 'Fetch the source %s after logging into %s' % (sources[0], homepage)
+
 builddependencies = [
     ('CMake', '3.27.6'),
     ('binutils', '2.40'),
 ]
 
+preconfigcmds = 'mkdir -p %(installdir)/bin'
+
```

Comment:
```suggestion
```
No need, it is done in the `postinstallcmds`

Reply:
of course! what have I been thinking? Fixed the clutter in 05d035dc78